mds/migrations/0028_auto_20190618_1352.py

```diff
@@ -0,0 +1,32 @@
+# Generated by Django 2.1.3 on 2019-06-18 13:52
+
+import django.contrib.postgres.functions
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [("mds", "0027_eventrecord_first_saved_at")]
+
+    operations = [
+        migrations.SeparateDatabaseAndState(
```

Comment:
What is the difference with two sequential operations? Or even two migrations with their own transaction each?